<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CartRow</class>
 <widget class="QWidget" name="CartRow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>70</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>70</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_row">
     <property name="styleSheet">
      <string notr="true">
        QFrame#frame_row {
            background-color: #F8FAFC;
            border: 1px solid #E2E8F0;
            border-radius: 8px;
        }
      </string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="info_layout">
        <item>
         <widget class="QLabel" name="lbl_product_name">
          <property name="styleSheet">
           <string>font-weight: bold; color: #1E293B; border:none;</string>
          </property>
          <property name="text">
           <string>Product Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_price">
          <property name="styleSheet">
           <string>color: #64748B; font-size: 12px; border:none;</string>
          </property>
          <property name="text">
           <string>P0.00</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="frame_qty">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string>background-color: white; border-radius: 4px; border: 1px solid #CBD5E1;</string>
        </property>
        <property name="fixedSize" stdset="0">
         <size>
          <width>90</width>
          <height>30</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="layout_qty">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin" stdset="0">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="btn_minus">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string>border:none; font-weight:bold; color: #64748B;</string>
           </property>
           <property name="text">
            <string>-</string>
           </property>
           <property name="fixedSize" stdset="0">
            <size>
             <width>25</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_qty">
           <property name="styleSheet">
            <string>border:none; font-weight: 800; color:#0F172A;</string>
           </property>
           <property name="text">
            <string>1</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_plus">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string>border:none; font-weight:bold; color: #64748B;</string>
           </property>
           <property name="text">
            <string>+</string>
           </property>
           <property name="fixedSize" stdset="0">
            <size>
             <width>25</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_remove">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string>
             QPushButton { background: transparent; color: #EF4444; font-size: 18px; font-weight: bold; border: none; }
             QPushButton:hover { background-color: #FEF2F2; border-radius: 12px; }
          </string>
        </property>
        <property name="text">
         <string>Ã—</string>
        </property>
        <property name="fixedSize" stdset="0">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
